project/src/components/app/app.tsx

```diff
@@ -1,5 +1,13 @@
-function App(): JSX.Element {
-  return <p>Hello, world!</p>;
+import MainPage from '../../pages/main-page/main-page';
+
+type Props = {
```

Comment:
Все же общий паттерн это название компонента + Props
```suggestion
type AppProps = {
```